### DISCUSSION TITLE
Strange HTTP Traffic in Hunt

### DISCUSSION THREAD

**Discussion by LS3-Allen** [Security-Onion-Solutions/securityonion#7897] [Unsupported Versions] [answered]
I was doing some adhoc hunting today and I noticed a lot of HTTP events coming from many random public IP address, going to many random IP addresses on my network.  Most of the source IP addresses are located in Europe or Russia.  I drilled down to look at the details and see that they are GET http methods with a http.virtual_host of www.*****porn.com.

All of this activity is hitting port 80 on my servers.  I'm wondering if this is some kind of proxy like activity where the source IP address is using my servers to access pornography sites.  Am I correct in that thinking, and how could I dig further into it?

Here is an example of what I am seeing in Hunt:

<html>
<body>

```
@timestamp | 2022-05-02T17:38:36.336Z
@version | 1
client.ip | 91.201.241.41
client.port | 59562
destination.ip | 172.16.156.97
destination.port | 80
ecs.version | 1.8.0
event.category | network
event.dataset | http
event.module | zeek
http.method | GET
http.request.body.length | 0
http.response.body.length | 0
http.trans_depth | 1
http.uri | /
http.virtual_host | www.youporn.com
ingest.timestamp | 2022-05-02T17:38:43.238Z
log.file.path | /nsm/zeek/logs/current/http.log
log.id.uid | CMzep31uafoGsADqSj
log.offset | 21047072
message | {"ts":1651513116.336492,"uid":"CMzep31uafoGsADqSj","id.orig_h":"91.201.241.41","id.orig_p":59562,"id.resp_h":"172.16.156.97","id.resp_p":80,"trans_depth":1,"method":"GET","host":"www.youporn.com","uri":"/","request_body_len":0,"response_body_len":0,"tags":[]}
```

**Comment by dougburks** [comment, marked as answer]
> http.request.body.length | 0
> http.response.body.length | 0

Looks like not much happened. You might also look for the `http.status_code` and `http.status_message` fields or try pivoting to PCAP to see the full TCP stream.
